libr/core/disasm.c

```diff
@@ -1232,6 +1233,21 @@ static void ds_newline(RDisasmState *ds) {
 	}
 }
 
+static void ds_begin_cont(RDisasmState *ds) {
+	ds_begin_line (ds);
+	ds_setup_print_pre (ds, false, false);
+	if (!ds->linesright && ds->show_lines_bb && ds->line) {
+		RAnalRefStr *refstr = r_anal_reflines_str (ds->core, ds->at,
+		                    ds->linesopts | R_ANAL_REFLINE_TYPE_MIDDLE_AFTER);
+		char *refline = refstr->str;
+		char *reflinecol = refstr->cols;
+		ds_print_ref_lines (refline, reflinecol, ds);
```

Comment:
What about making ranalreflinestr.free() instead of freeing internal struct fields by hand?

Reply:
Ok b73cc83c76269d02b8be05ff24ec880baf69569b